### PR TITLE
Added type specifications to appointments and grades.

### LINKED ISSUE #117: Clarify the meaning of types
Just like this code block in `absenceInfo.js`:
https://github.com/simplyGits/MagisterJS/blob/2087f489d6a850943d31e49932935c0259fb6062/src/absenceInfo.js#L68-L79

I would suggest implementing or documenting the same for other instances such as appointment info types. Below I gathered a list of some of the useful types I could find.

AfspraakInfoType:
0: "None"
1: "Huiswerk"
2: "Proefwerk"
3: "Tentamen"
4: "SchriftelijkeOverhoring"
5: "MondelingeOverhoring"
6: "Informatie"
7: "Aantekening"

AfspraakStatus:
0: "GeenStatus"
1: "GeroosterdAutomatisch"
2: "GeroosterdHandmatig"
3: "Gewijzigd"
4: "VervallenHandmatig"
5: "VervallenAutomatisch"
6: "InGebruik"
7: "Afgesloten"
8: "Ingezet"
9: "Verplaatst"
10: "GewijzigdEnVerplaatst"

AfspraakType:
0: "None"
1: "Persoonlijk"
2: "Algemeen"
3: "Schoolbreed"
4: "Stage"
5: "Intake"
6: "Roostervrij"
7: "Kwt"
8: "Standby"
9: "Blokkade"
10: "Overig"
11: "BlokkadeLokaal"
12: "BlokkadeKlas"
13: "Les"
14: "Studiehuis"
15: "RoostervrijeStudie"
16: "Planning"
101: "Maatregelen"
102: "Presenties"
103: "ExamenRooster"

Kolomsoort:
0: "Unknown"
1: "Cijfer"
2: "Gemiddelde"
3: "Maximum"
4: "Formule"
5: "Minimum"
6: "Som"
7: "Tel"
8: "Cevo"
9: "VrijeTekst"
10: "CevoCPE"
11: "CevoCIE"
12: "Weegfactor"
13: "Eindcijfer"
14: "Tekortpunten"
15: "BoomTop"
16: "VakVoorwaarde"

### PR DESCRIPTION
**Pull request by Argetan**
The title pretty much says it all, however, I was unable to test the implemented functionality due to me not having access to a Magister account anymore so if somebody could confirm it's working properly that would be great 😃 

This PR will close #117.

### CODE REVIEW

File: src/appointment.js [hunk 1]
```diff
@@ -142,6 +142,78 @@ class Appointment extends MagisterThing {
...
+	 * @type {String}
+	 * @readonly
+	 */
+	get type() {
+		switch (this.type) {
+			case 0:   return 'none' // None
+			case 1:   return 'personal' // Persoonlijk
+			case 2:   return 'general' // Algemeen
+			case 3:   return 'school wide' // School breed
+			case 4:   return 'stage' // Stage
```
Comment (lieuwex) [inline src/appointment.js]: I think this should be internship?

Comment (Argetan) [reply src/appointment.js]: Jup good catch 😅

**Comment by Argetan** [comment]
> Pretty good, but can you change the indentation back to what it was [here](https://github.com/simplyGits/MagisterJS/pull/131/files#diff-8172ed8ff68ac1202f8cda40b9a3c008L81) for example? (so no extra indentation level in switch blocks)

Done 😃

**Comment by lieuwex** [comment]
Thanks!

**Comment by lieuwex** [comment]
Oh no! I catched a bug earlier but forgot about it when I went to write the review. I will fix it on master.

**Pull request merged**